erpnext_com/www/docs/user/manual/en/setting-up/data/personal-data-deletion.md

```diff
@@ -0,0 +1,46 @@
+<!--add breadcrumbs-->
+
+# Personal Data Deletion
+
+Personal data deletion tool enables a user to delete all the personal data a user has generated while using ERPNext.
+
+## Request to Delete Data Webform
+
+To begin deleting data, you need to visit [host-name]/request-to-delete-data (e.g. example.erpnext.com/request-to-delete-data) in the url field.
+
+<img class="screenshot" alt="Request Data Webform" src="{{docs_base_url}}/assets/img/setup/personal-data-deletion-request/request-to-delete-data-webform.png">
+
+After submitting your request, you will receive a success response.
+
+<img class="screenshot" alt="Deletion Request Success" src="{{docs_base_url}}/assets/img/setup/personal-data-deletion-request/deletion-request-success.png">
+
+## Email Verification
+
+This will sent a mail with a verification link of the data to the email address associated with the user.
+
+<img class="screenshot" alt="Verification Email" src="{{docs_base_url}}/assets/img/setup/personal-data-deletion-request/verification-email.png">
+
+Once the user clicks on the verification link.
+<img class="screenshot" alt="Confirmed Verification" src="{{docs_base_url}}/assets/img/setup/personal-data-deletion-request/confirmed-verification.png">
+
+## Personal Data Delete Request DocType
```

Comment:
line 26 could be 2. How it works or something similar and the subsequent steps can be 2.1... etc
you don't need the escape character "\" when text is a heading